app/src/main/java/org/feature/fox/coffee_counter/ui/authentication/AuthenticationViewModel.kt

```diff
@@ -76,8 +78,20 @@ class AuthenticationViewModel @Inject constructor(
         if (response.data == null) {
             toastMessage.value = response.message ?: resource.getString(R.string.unknown_error)
             return
+        } else {
+            userRepository.insertUser(
+                User(
+                    id = idState.value.text,
+                    name = nameState.value.text,
+                    false,
+                    password = BCrypt.withDefaults()
+                        .hashToString(12, passwordState.value.text.toCharArray())
+                )
+            )
```

Comment:
No need for an else case if the `if` already has a return inside.

Reply:
Should be resolved with ec8b6109016cacdaa8ebd4489789fc55f978eb85